src/payment_import/fio_cz/manifest.json

```diff
@@ -0,0 +1,35 @@
+{
+  "version": "1",
+  "information": {
+    "name": "fio_cz-import",
+    "displayName": "Fio bank payment import",
+    "description": "This plugin enables you to import payments from Fio bank to UCRM. See https://www.fio.cz/bankovni-sluzby/api-bankovnictvi",
+    "url": "https://github.com/Ubiquiti-App/UCRM-plugins",
+    "version": "1.0.0",
+    "ucrmVersionCompliancy": {
+      "min": "2.10.0-dev",
+      "max": null
+    },
+    "author": "Ubiquiti Networks, Inc."
+  },
+  "configuration": [
+    {
+      "key": "token",
+      "label": "Fio token",
+      "description": "Token obtained from Fio bank internet banking.",
+      "required": 1
+    },
+    {
+      "key": "startDate",
+      "label": "Payments start date",
+      "description": "Payments older than this date will be ignored.Use d.m.Y format",
+      "required": 1
+    },
+    {
+      "key": "paymentMatchAttribute",
+      "label": "Match attribute from payment variable symbol to UCRM",
+      "description": "Can be 'invoiceNumber', 'clientId', 'clientUserIdent' or a custom attribute key.",
+      "required": 1
```

Comment:
since you don't have `lastProcessedPayment` defined here, it will be deleted from `config.json`